composer.json

```diff
@@ -95,7 +95,8 @@
         "doctrine/mongodb-odm": "<2.4",
         "doctrine/persistence": "<1.3",
         "symfony/var-exporter" : "<6.1.1",
-        "phpunit/phpunit": "<9.5",
+		"phpunit/phpunit": "<9.5",
```

Comment:
This should work even without this line.

Reply:
yes mb I was doing many things at once ^^

Reply:
actually we need this for our lowest tests I'll probably move this to the ci though